qbsp/brush.cc

```diff
@@ -382,11 +383,11 @@ CreateBrushFaces(hullbrush_t *hullbrush, const vec3_t rotate_offset,
 
     mapface = hullbrush->faces;
     for (i = 0; i < hullbrush->numfaces; i++, mapface++) {
-        if (!hullnum) {
+        if (!hullnum && hullbrush->contents == CONTENTS_HINT) {
             /* Don't generate hintskip faces */
             const mtexinfo_t &texinfo = map.mtexinfos.at(mapface->texinfo);
             const char *texname = map.miptex.at(texinfo.miptex).c_str();
-            if (!Q_strcasecmp(texname, "hintskip"))
+            if (Q_strcasecmp(texname, "hint"))
```

Comment:
I think this line was correct before. "hintskip" faces from the hullbrush_t should be skipped over with `continue;`, the other faces in a hint brush should have the "hint" texture and these need to be processed normally and go into the facelist returned from CreateBrushFaces.

Reply:
oh - I get it, it's just treating anything other than "hint" like the old code used to treat "hintskip". Looks good then.